workflow-templates/jenkins-security-scan.yaml

```
@@ -0,0 +1,15 @@
+name: Jenkins Security Scan
+on:
+  push:
+    branches:
+      - $default-branch
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+  workflow_dispatch:
+
+jobs:
+  security-scan:
+    uses: jenkins-infra/jenkins-security-scan/.github/workflows/jenkins-security-scan.yaml@v1
+    with:
+      java-cache: '' # Optionally enable use of a build dependency cache. Specify 'maven' or 'gradle' as appropriate.
+      java-version: 8 # What version of Java to set up for the build.
```

```suggestion
      java-version: 11 # What version of Java to set up for the build.
```

Given https://github.com/jenkinsci/plugin-pom/pull/478 it probably makes sense to have the template default to `11` here.